include/openPMD/ReadIterations.hpp

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2021 Franz Poeschel
+ *
+ * This file is part of openPMD-api.
+ *
+ * openPMD-api is free software: you can redistribute it and/or modify
+ * it under the terms of of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * openPMD-api is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with openPMD-api.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "openPMD/Iteration.hpp"
+#include "openPMD/Series.hpp"
+
+/** Writing side of the streaming API.
```

Comment:
If you add a doxygen block like this here, it will "pop down" to the next best match, which is `namespace openPMD`.

Do you like to make this a block for the file itself? In that case do this:
```suggestion
/** @file
 *
 * Writing side of the streaming API.
```

Reply:
Copy paste error, that's not supposed to be there. Thanks for noticing.